### DISCUSSION TITLE
student application

### DISCUSSION THREAD

**Discussion by tzur320** [community/community#118398] [GitHub Education]
### Hi everyone,

hello!
how much time after approval of request do i need to wait to get the benefits of educational package?

**Comment by arafajs** [comment]
In most cases, you can expect to receive your benefits from the GitHub Student Developer Pack within  3-7 business days after your application is approved.

**Comment by jfullstackdev** [comment]
as for me, after completing all the steps, I was approved within 3 hours from the time I submitted it for Staff approval and making it approve and go live

**Reply by ghost** [reply]
正确

**Comment by samus-aran** [comment]
Hi @tzur320 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
